presto-main/src/test/java/com/facebook/presto/type/TestFloatOperators.java

```diff
@@ -31,4 +31,68 @@ public void testTypeConstructor()
         assertFunction("FLOAT'Infinity'", FLOAT, Float.POSITIVE_INFINITY);
         assertFunction("FLOAT'-Infinity'", FLOAT, Float.NEGATIVE_INFINITY);
     }
+
+    @Test
+    public void testAdd()
```

Comment:
This is because of https://github.com/prestodb/presto/pull/5257#discussion_r63769537. I don't expect any special handling of `Float.MAX_VALUE` + 1 or `MAX_VALUE` + `MAX_VALUE`, it's IEEE754 specifics and I believe Java is doing it well. ;)